in_text/text_02_driver.yml

```diff
@@ -0,0 +1,730 @@
+        data-units: integer
+      -
+        attr-label: SHTFL
+        attr-def: >-
+           Sensible Heat Net Flux
+        attr-defs: This data release
+        data-min: NA
+        data-max: NA
+        data-units: W/m^2
+      -
+        attr-label: SNOD
+        attr-def: >-
+           Snow Depth
+        attr-defs: This data release
+        data-min: NA
+        data-max: NA
+        data-units: m
+      -
+        attr-label: UGRD
+        attr-def: >-
+           U-Component of Wind
+        attr-defs: This data release
+        data-min: NA
+        data-max: NA
+        data-units: m/s
+      -
+        attr-label: VGRD
+        attr-def: >-
+           V-Component of Wind
+        attr-defs: This data release
+        data-min: NA
+        data-max: NA
+        data-units: m;/s
+      -
+        attr-label: WEASD
+        attr-def: >-
+           Water Equivalent of Accumulated Snow Depth
+        attr-defs: This data release
+        data-min: NA
+        data-max: NA
+        data-units: kg/m^2
+  -
+    data-name: reservoir_io_obs.csv
+    data-description: >-
+      Daily water temperature observations for inflow and outflow reaches of the Pepacton, Cannonsville, and Neversink reservoirs.
+    
+    attributes:
+    -
+      attr-label: res_id
+      attr-def: >-
+        Unique reservoir identification for this dataset, which comes from the Prmnn_I from NHD high-res prefixed with source, as "nhdhr_{Prmnn_I}".
+        
+        nhdhr_120022743 is Cannonsville Reservoir and nhdhr_151957878 is Pepacton Reservoir 
+      attr-defs: >- 
+        http://nhd.usgs.gov/
+      data-min: NA
+      data-max: NA
+      data-units: NA
+    -
+      attr-label: site_id
+      attr-def: >-
+        Site identifier from the data provider.
+      attr-defs: This data release
+      data-min: NA
+      data-max: NA
+      data-units: NA
+    -
+      attr-label: date
+      attr-def: >-
+        Date of discharge and temperature observations
+      attr-defs: This data release
+      data-min: 19341001
+      data-max: 20201217
+      data-units: NA
+    -
+      attr-label: flow_cms
+      attr-def: >-
+        Discharge, in cubic meters per second
+      attr-defs: This data release
+      data-min: 0.002831685
+      data-max: 784.3767
+      data-units: cubic meters per second
+    -
+      attr-label: mean_temp_degC
+      attr-def: >-
+        Mean daily observed temperature
+      attr-defs: This data release
+      data-min: 0.1
+      data-max: 27
+      data-units: degrees C
+    -
+      attr-label: min_temp_degC
+      attr-def: >-
+        Minimum daily observed temperature
+      attr-defs: This data release
+      data-min: 0.1
+      data-max: 30.5
+      data-units: degrees C
+    -
+      attr-label: max_temp_degC
+      attr-def: >-
+        Maximum daily observed temperature
+      attr-defs: This data release
+      data-min: 0.1
+      data-max: 30.5
+      data-units: degrees C
+    -
+      attr-label: flag
```

Comment:
@limnoliver  Do we know why this attribute exists? It was not documented in last year's data release.

Reply:
incorporated text from main to this entity.